test/TestApps/ToastNotificationsDemoApp/main.cpp

```diff
@@ -136,24 +193,9 @@ int main()
     // Display the current AppNotificationSetting for the app.
     std::wcout << L"Printing AppNotificationSetting for app: " << GetEnumString(appNotificationManager.Setting()) << "\n\n";
 
-    winrt::AppNotificationActivationInfo activationInfo{ nullptr };
-    // Create toastActivationInfo depending on packaged | unpackaged scenario.
-    if (isPackaged)
-    {
-        std::wcout << L"Calling AppNotificationActivationInfo with ToastActivatorCLSID in manifest...\n";
-        activationInfo = winrt::AppNotificationActivationInfo(winrt::guid("FE8C7374-A28F-4CBE-8D28-4288CBDFD431"));
-        std::wcout << L"Done.\n\n";
-    }
-    else
-    {
-        std::wcout << L"Calling AppNotificationActivationInfo with app assets...\n";
-        activationInfo = winrt::AppNotificationActivationInfo(L"AppNotificationApp", winrt::Uri{ LR"(C:\Windows\System32\WindowsSecurityIcon.png)" });
-        std::wcout << L"Done.\n\n";
-    }
-
     // Registering app for activation
-    std::wcout << L"Calling AppNotificationActivationInfo::Register(activationInfo)...\n";
-    appNotificationManager.Register(activationInfo);
+    std::wcout << L"Calling AppNotificationManager::Register()...\n";
```

Comment:
Shouldn't FG activator be called before REgister @pmpurifoy ?

Reply:
Yes